components/en/interview_cards.html

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
This file was auto generated by our build.js, from the content provided in one of the json files. If we would like to change any content of the cards we are to do so in the json files. If we would like to change something in the html code, than build.js it is, more specifically class InterviewCards and function getCode(). This is the product of my masterpiece.

Reply:
in that case what's the point of committing it? instead it could be just be added to .gitignore so that someone doesn't accidentally edit it and then all their edits disappear

Reply:
It's a component and by removing it, I could potentially disrupt the structure.